modules/imgcodecs/include/opencv2/imgcodecs.hpp

```diff
@@ -405,14 +405,28 @@ The function imencode compresses the image and stores it in the memory buffer th
 result. See cv::imwrite for the list of supported formats and flags description.
 
 @param ext File extension that defines the output format. Must include a leading period.
-@param img Image to be written.
+@param img Image to be compressed.
```

Comment:
(Mat or vector of Mat) Image or Images to be compressed.

@asmorkalov imwrite() writes single and multiple images. AFAIK imwritemulti() is just for python. also imencode() behaves same as imwrite()

Reply:
i think imwritemulti() also be defined as CV_EXPORTS_W and needs documentation